tools/e2e/run_e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+# Kill any processes on exit
+trap 'kill $(jobs -p)' EXIT
+
+GiteaFlags=()
+
+[[ -v GITEA_CUSTOM ]] && GiteaFlags+=(-C "${GITEA_CUSTOM}")
```

Comment:
The built-in bash on macOS is ancient.  When I run `make test-e2e`, I get this:

```
./tools/e2e/run_e2e.sh: line 9: conditional binary operator expected
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
make: *** [test-e2e-sqlite] Error 1
```

It should either use this shebang:

```
#!/usr/bin/env bash
```

... or use more compatible syntax.  The latter will probably lead to fewer gotchas.